### PR TITLE
Edit user guide

### PR DESCRIPTION
**Pull request by C-likethis123**
* Tone changed to a more conversational one
* Include labeled diagrams with captions
* Add more preface sentences before each section

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -291,46 +320,42 @@ Deletes a person with the NRIC S98654322D in ORGANice.
...
+kidney matching tests on a specific patient, refer to <<Matching a specific patient>>.
 
 ==== Matching a specific patient
-For example, a user wants to match a specific patient called "John Doe" and his NRIC is "S1234568R".
+This section will explain how to run kidney matching tests on a specific patient and run you through an example.
+To run a match of a specific patient, type in a command of the format `match ic/**NRIC**`.
 
-. Type `match ic/S1234568R` in the command box and press Enter.
+Here is an example. Suppose you want to match a specific patient called "John Doe" and his NRIC is "S1234568R". This is
+what to do:
```
Comment (Yen-Peng) [inline docs/UserGuide.adoc]: Perhaps you can use "This is what **you can** do:" to make it sound more conversational.

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -35,26 +35,48 @@ to get started.
...
+1.  Ensure you have Java `11` or above installed in your computer. +
 If not, install it link:{javaURL}[here].
-.  Download the latest `organice.jar` link:{repoURL}/releases[here].
-.  Copy the jar file to the folder you want to use as the home folder for your application.
-.  Double-click the file to start the application. The application should appear in a few seconds.
-The following screenshot is a sample of how the application looks like:
-+
+2.  Download the latest `organice.jar` link:{repoURL}/releases[here].
+3.  Copy the jar file to the folder you want to use as the home folder for your application.
+4.  Double-click the file to start the application. The application should appear in a few seconds.
```
Comment (SimNee) [inline docs/UserGuide.adoc]: Double-click the JAR file may sounds a bit easier to understand for slow user

Comment (C-likethis123) [reply docs/UserGuide.adoc]: Amended.

File: docs/UserGuide.adoc (hunk 1 above)
Comment (C-likethis123) [reply docs/UserGuide.adoc]: Amended.

**Pull request merged**